pos_partner_location_abstract/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+from odoo import fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    qr_code_url = fields.Char(compute="_compute_qr_code_url")
+
+    def _compute_qr_code_url(self):
+        """Compute QR Code URL for map provider"""
+        self.write({"qr_code_url": ""})
```

Comment:
This is a non stored field. So using "write" will do no benefit here because no db tables are updated. 
You should use "update" instead.

Reply:
Agree with you, applied proposed changes.